cards/src/main/resources/cards/custom/group0/other_death_knight/oldgods/minion_saronite_drake.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Saronite Drake",
+  "baseManaCost": 5,
+  "type": "MINION",
+  "heroClass": "SPIRIT",
+  "baseAttack": 4,
+  "baseHp": 7,
+  "rarity": "RARE",
+  "race": "DRAGON",
+  "description": "Battlecry and Deathrattle: Duplicate the top card of your deck.",
```

Comment:
`Battlecry and Deathrattle: Copy the top card of your deck. Put it on top of your deck.` maybe. Duplicate isn't used often and it isn't obvious it puts it on top again.